src/java.base/share/classes/java/util/Collections.java

```diff
@@ -2653,6 +2765,12 @@ public NavigableSet<E> tailSet(E fromElement, boolean inclusive) {
                 return new SynchronizedNavigableSet<>(ns.tailSet(fromElement, inclusive), mutex);
             }
         }
+
```

Comment:
Is this `SynchronizedNavigableSet` missing overrides for the `addFirst/addLast/removeFirst/removeLast/getFirst/getLast` default methods?